.github/workflows/pr.yaml

```diff
@@ -42,7 +44,7 @@ jobs:
 
     - name: Deploy the chart
       run: |
-        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace $GITHUB_ACTOR
+        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace default
```

Comment:
I recommend use `${{ github.run_number }}` as part of namespace and chart name. So, we can have several PR actions in parallel without name conflicts

Reply:
Have updated the issue #11 to bring that in at a later date.